test/test_config_shinken.py

```diff
@@ -0,0 +1,126 @@
+            u'Guessing the property use_ssl type because it is not in Config object properties',
+            u'Host graphite use/inherit from an unknown template: graphite ! from: cfg/_shinken/hosts/graphite.cfg:1',
+            u'Guessing the property hostgroup_name type because it is not in Escalation object properties',
+            u'Guessing the property direct_routing type because it is not in ReceiverLink object properties'
+        ]
+
+        # Arbiter named as in the configuration
+        assert self.arbiter.conf.conf_is_correct
+        arbiter_link = self.arbiter.conf.arbiters.find_by_name('arbiter-master')
+        assert arbiter_link is not None
+        assert arbiter_link.configuration_errors == []
+        assert arbiter_link.configuration_warnings == []
+
+        # Scheduler named as in the configuration
+        assert self.arbiter.conf.conf_is_correct
+        scheduler_link = self.arbiter.conf.schedulers.find_by_name('scheduler-master')
+        assert scheduler_link is not None
+        # Scheduler configuration is ok
+        assert self.schedulers['scheduler-master'].sched.conf.conf_is_correct
+
+        # Broker, Poller, Reactionner named as in the configuration
+        link = self.arbiter.conf.brokers.find_by_name('broker-master')
+        assert link is not None
+        link = self.arbiter.conf.pollers.find_by_name('poller-master')
+        assert link is not None
+        link = self.arbiter.conf.reactionners.find_by_name('reactionner-master')
+        assert link is not None
+
+        # Receiver - no default receiver created
+        link = self.arbiter.conf.receivers.find_by_name('receiver-master')
+        assert link is not None
+
+        for item in self.arbiter.conf.commands:
+            print("Command: %s" % item)
+        assert len(self.arbiter.conf.commands) == 106
+
+        for item in self.arbiter.conf.timeperiods:
+            print("Timeperiod: %s" % item)
+        assert len(self.arbiter.conf.timeperiods) == 4
+
+        for item in self.arbiter.conf.contacts:
+            print("Contact: %s" % item)
+        assert len(self.arbiter.conf.contacts) == 7
+
+        for item in self.arbiter.conf.contactgroups:
+            print("Contacts group: %s" % item)
+        assert len(self.arbiter.conf.contactgroups) == 3
+
+        for item in self.arbiter.conf.hosts:
+            print("Host: %s" % item)
+        assert len(self.arbiter.conf.hosts) == 13
+
+        for item in self.arbiter.conf.hostgroups:
+            print("Hosts group: %s" % item)
+        assert len(self.arbiter.conf.hostgroups) == 8
+
+        for item in self.arbiter.conf.services:
+            print("Service: %s" % item)
+        assert len(self.arbiter.conf.services) == 94
+
+        for item in self.arbiter.conf.servicegroups:
+            print("Services group: %s" % item)
+        assert len(self.arbiter.conf.servicegroups) == 5
```

Comment:
Can you test the services properties, I think for `Cpu` for example because it be defined 2 times but with not the same template, so need to test each has the right properties.